docs/guides/phoenix_walkthrough.md

```diff
@@ -42,8 +42,8 @@ to the following:
 
 ```elixir
 config :phoenix_distillery, PhoenixDistilleryWeb.Endpoint,
-  http: [:inet6, port: System.get_env("PORT") || 4000],
-  url: [host: "localhost", port: System.get_env("PORT")], # This is critical for ensuring web-sockets properly authorize.
+  http: [:inet6, port: {:system, "PORT"} || 4000],
```

Comment:
I believe usage of the `:system` tuple is discouraged and probably deprecated, I'd prefer to leave the usage of `get_env` instead